apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -3,72 +3,130 @@
 
 {% block content %}
 <div class="course-detail">
-
   <h1 class="course-detail__title">{{ current_page.get_title }}</h1>
 
   {% with current_page.course as course %}
 
-    <ul class="course-detail__subjects">
-      {% for subject in course.subjects.all %}
-        {# If the current page is a draft, show draft subjects with a class annotation for styling #}
-        {% if current_page.publisher_is_draft %}
-          {% if subject.public_extension %}
-            <li class="course-detail__subjects__item">
-              {{ subject.public_extension.extended_object.get_title }}
-            </li>
-          {% else %}
-            <li class="course-detail__subjects__item--draft">
-              {{ subject.extended_object.get_title }}
-            </li>
-          {% endif %}
-        {# If the current course page is the published version, show only the subjects that are published #}
-        {% elif subject.public_extension %}
-          <li class="course-detail__subjects__item">
-            {{ subject.public_extension.extended_object.get_title }}
-          </li>
-        {% endif %}
-      {% empty %}
-        <li class="course-detail__subjects--empty">
-          {% trans "No associated subjects" %}
-        </li>
-      {% endfor %}
-    </ul>
-
-    <ul class="course-detail__organizations">
-      {% for organization in course.organizations.all %}
-        {# If the current page is a draft, show draft organizations with a class annotation for styling #}
-        {% if current_page.publisher_is_draft %}
-          {% if organization.public_extension %}
-            <li class="course-detail__organizations__item">
-              {{ organization.public_extension.extended_object.get_title }}
-            </li>
-          {% else %}
-            <li class="course-detail__organizations__item--draft">
-              {{ organization.extended_object.get_title }}
+  <div class="course-detail__teaser">
+    {% placeholder "course_teaser" %}
+  </div>
+
+  <div class="course-detail__content">
+
+    <div class="course-detail__content__syllabus">
+      <h2>{% trans 'About the course' %}</h2>
+      {% with header_level=3 %}
+        {% placeholder "course_syllabus" or %}
+        <p>{% trans 'Required' %}</p>
+        {% endplaceholder %}
+      {% endwith %}
+    </div>
+
+    <div class="course-detail__content__format">
+      <h2>{% trans 'Format' %}</h2>
+      {% placeholder "course_format" or %}
+      <p>{% trans 'Required' %}</p>
```

Comment:
How is the course structured?

Reply:
Is this another default content for empty content ?

Reply:
For empty content. But actually I think you forgot the {% endplaceholder %} closing tag.